electrical/motor_controller.py

```diff
@@ -1,20 +1,22 @@
 import time
 #from engine.robot import Robot
-if False:  # change to True when running code on robot
```

Comment:
I think you can add this back. might help with failing check

Reply:
Good point - added it back